src/Package/Impl/Microsoft.VisualStudio.R.Package.csproj

```diff
@@ -861,11 +867,11 @@
       <HintPath>$(LibDirectory)\Microsoft.VisualStudio.InteractiveWindow.dll</HintPath>
     </Reference>
     <Reference Include="Microsoft.VisualStudio.Language.Intellisense, Version=14.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>..\..\..\NugetPackages\Microsoft.VisualStudio.Language.Intellisense.14.1.24720\lib\net45\Microsoft.VisualStudio.Language.Intellisense.dll</HintPath>
+      <HintPath>..\..\..\NugetPackages\Microsoft.VisualStudio.Language.Intellisense.$(Vs14Version)\lib\net45\Microsoft.VisualStudio.Language.Intellisense.dll</HintPath>
```

Comment:
It doesn't really. I think I may just put back version number back. Generally versioning issue happen around CPS related code that is in the project assembly, package and 3 test modules. So working with nuget in these component has to be careful and project files may need to be fixed if nuget references are added/removed/updated.